modules/ROOT/pages/apikit-for-asyncapi-supported-bindings.adoc

```diff
@@ -3,41 +3,148 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-APIkit for AsyncAPI supports various bindings to enable message-driven architectures.
+APIkit for AsyncAPI supports various bindings to enable message-driven architectures, and parametrized channels to define dynamic and reusable configurations.
 
-[%header%autowidthspread]
+== Supported Bindings
+
+=== Anypoint Connector for MQ (Anypoint MQ Connector) 
+
+[%header%autowidthspread,cols="1a,1a,1a,1a,1a"]
+|===
+| Binding | Publish | Message Listener | Description | Reference
+
+| `Destination`
+| `Destination`
+| `Destination`
+| `Destination` (queue or exchange) name for this channel. It's optional and defaults to the channel name. Specify it if the channel name differs from the actual `destination` name. For example, if the channel name isn't a valid `destination` name in Anypoint MQ.
+.2+| See https://github.com/asyncapi/bindings/blob/b1e7736071ccddc6c3d652bb2071cad4549f8a08/anypointmq/README.md[Anypoint MQ Bindings].
+
+|`DestinationType`
+|`DestinationType`
+|`DestinationType`
+| `DestinationType` (queue or exchange) name for this channel. It's optional and defaults to `queue`. Specify it if `exchange` is needed instead of a `queue`.
```

Comment:
```suggestion
| Type of destination (queue or exchange, or FIFO queue). It's optional and defaults to `queue`. Specifies the messaging model supported by this channel:

* Publish/subscribe
* Point-to-point
* Strict message ordering
```

Reply:
@ashutoshsharmasf, can you review and approve this PR if the changes are OK?
Thanks!